web/tasks/Wiki-Auto.md

```diff
@@ -1,119 +1,213 @@
+- `simple_sentence`: a sentence from Simple English Wikipedia.
+- `simple_sentence_id`: a unique ID for each Simple English Wikipedia sentence. The last two dash-separated numbers correspond to the paragraph number in the article and the sentence number in the paragraph.
+- `alignment_label`: signifies whether a pair of sentences is aligned: labels are `>=1:aligned` and `0:notAligned`
+- `paragraph_alignment`: a first step of alignment mapping English and Simple English paragraphs from linked articles
+- `sentence_alignment`: the full alignment mapping English and Simple English sentences from linked articles
 
 ### Data Statistics
 
-[More Information Needed]
+In `auto`, the `part_2` split corresponds to the articles used in `manual`, and `part_1` has the rest of Wikipedia.
+
+The `manual` config is provided with a `train`/`dev`/`test` split with the following amounts of data:
+|                            | Tain   | Dev   | Test   |
+| -----                      | ------ | ----- | ----   |
+| Total sentence pairs       | 373801 | 73249 | 118074 |
+| Aligned sentence pairs     |  1889  |  346  | 677    |
+
+The `auto_acl` has 488,332 complex-sentence pairs that are to be used for training the model. The average sentence length for complex and simple sentences are 26.6 and 18.7 respectively.
 
 ## Dataset Creation
 
 ### Curation Rationale
 
-[More Information Needed]
+Wiki-Auto provides a new version of the Wikipedia corpus that is larger, contains 75% less defective pairs and has more complex rewrites than the previous WIKILARGE dataset.  
 
 ### Communicative Goal
 
-[More Information Needed]
-
 
 ### Source Data
 
 #### Initial Data Collection and Normalization
 
-[More Information Needed]
+The authors mention that they "extracted 138,095 article pairs from the 2019/09 Wikipedia dump using an improved version of the [WikiExtractor](https://github.com/attardi/wikiextractor) library". The [SpaCy](https://spacy.io/) library is used for sentence splitting.
 
 #### Who are the source language producers?
 
-[More Information Needed]
+The dataset uses language from Wikipedia: some demographic information is provided [here](https://en.wikipedia.org/wiki/Wikipedia:Who_writes_Wikipedia%3F).
 
 ### Annotations
 
 #### Annotation process
 
-[More Information Needed]
+Sentence alignment labels were crowdsourced for 500 randomly sampled document pairs (10,123 sentence pairs total). The authors pre-selected several alignment candidates from English Wikipedia for each Simple Wikipedia sentence based on various similarity metrics, then asked the crowd-workers to annotate these pairs. Finally, they trained their alignment model on this manually annotated dataset to obtain automatically aligned sentences (138,095 document pairs, 488,332 sentence pairs).
 
 #### Who are the annotators?
 
-[More Information Needed]
+No demographic annotation is provided for the crowd workers.
```

Comment:
Is it mentioned what platform was used to annotate?

Reply:
Added the platform used.